spec/acceptance/init_spec.rb

```diff
@@ -1,67 +1,102 @@
 # run a test task
 require 'spec_helper_acceptance'
-# bolt regexes
-# expect_multiple_regexes(result: result, regexes: [%r{"status":"(stopped|in_sync)"}, %r{Ran on 1 node}])
-# expect_multiple_regexes(result: result, regexes: [%r{"status":"stopped"}, %r{"enabled":"false"}, %r{Ran on 1 node}])
-
-def run_and_expect(params, regex_hash)
-  expect_multiple_regexes(result: run_task(task_name: 'service', params: params), regexes: regex_hash)
-end
+require 'beaker-task_helper/inventory'
+require 'bolt_spec/run'
 
 describe 'service task' do
+  include Beaker::TaskHelper::Inventory
+  include BoltSpec::Run
+
+  let(:module_path) { RSpec.configuration.module_path }
+  let(:config) { { 'modulepath' => module_path } }
+  let(:inventory) { hosts_to_inventory.merge('features' => ['puppet-agent']) }
+
+  def run(params)
+    run_task('service', 'default', params, config: config, inventory: inventory)
```

Comment:
Failing on windows because of using something defined with `let`in a before hook. 
https://jenkins-master-prod-1.delivery.puppetlabs.net/view/All/job/forge-module_puppetlabs-service_intn-sys_smoke-adhoc/PLATFORM=windows2012r2-64default.a,WORKER_LABEL=beaker/13/console

Reply:
Ah, I guess that's why they were all functions.